saddle-core/src/main/scala/org/saddle/package.scala

```diff
@@ -455,4 +455,11 @@ package object saddle {
 
   }
 
+  /** Filling method for NA values. Non-sealed because could add more variants
+    * in the future.
+    */
+  // private constructor to prevent public extensions
+  abstract class FillMethod private[saddle] ()
```

Comment:
why not sealed trait?

Reply:
I thought it'd leave options open for more cases in next iterations. Adding variants in an enum type is considered an incompatible change.